gcp_project_setup.txt

```diff
@@ -26,6 +26,9 @@ gcloud iam service-accounts keys create \
 # FIXME: It'd be better to create a more restrictive role with just the
 # permissions we need, intead of using the pre-defined roles.
 
+# Create bucket for uploading images of openBSD and netBSD, bucket name
+# should be unique
+gsutil mb -p $GCP_PROJECT-bucket -b on gs://pg-ci-bucket
```

Comment:
This doesn't look right? Think you meant to change the end of the line, not the project name?

Reply:
My bad, I will update it.